liblouis/compileTranslationTable.c

```diff
@@ -884,7 +824,7 @@ addRule(FileInfo *nested, TranslationTableOpcode opcode, CharsString *ruleChars,
 		}
 	if (!nofor) {
 		if (rule->charslen == 1)
-			addForwardRuleWithSingleChar(nested, newRuleOffset, rule, *table);
+			addForwardRuleWithSingleChar(nested, newRuleOffset, rule, table);
```

Comment:
Kinda weird that addForwardRuleWithSingleChar takes a `table` and addForwardRuleWithMultipleChars takes a `*table`

Reply:
Maybe a bit weird, but correct.